DFC.ServiceTaxonomy.Editor/Views/ContentVisualise.Button.cshtml

```diff
@@ -18,6 +18,18 @@
        asp-controller="Visualise"
        asp-area="@typeof(DFC.ServiceTaxonomy.GraphVisualiser.Startup)!.Namespace"
        asp-route-visualise="@graphSyncPartText"
+       asp-route-graph="published"
        target="_blank"
-       class="publish-button btn btn-info">@T["Visualise (last published)"]</a>
+       class="publish-button btn @(!string.IsNullOrWhiteSpace(Model.ButtonSize) ? "btn-sm" : "") btn-info">@T["Visualise (last published)"]</a>
+}
+
+@if (graphSyncPartText != null)
+{
+    <a asp-action="@nameof(DFC.ServiceTaxonomy.GraphVisualiser.Controllers.VisualiseController.Viewer)"
+       asp-controller="Visualise"
+       asp-area="@typeof(DFC.ServiceTaxonomy.GraphVisualiser.Startup)!.Namespace"
+       asp-route-visualise="@graphSyncPartText"
+       asp-route-graph="preview"
+       target="_blank"
+       class="publish-button btn @(!string.IsNullOrWhiteSpace(Model.ButtonSize) ? "btn-sm" : "") btn-info">@T["Visualise (preview)"]</a>
```

Comment:
Same with the button size here, and change to "Visualise Preview"

Reply:
Done!